src/main/java/uk/gov/ch/repository/officers/ActiveOfficersDetailsRepository.java

```diff
@@ -33,14 +34,14 @@
             + "adr.PO_BOX as SERVICE_ADDRESS_PO_BOX, "
             + "adr.POST_CODE as SERVICE_ADDRESS_POST_CODE, "
             + "adr.REGION as SERVICE_ADDRESS_REGION, "
-            + "ura.HOUSE_NAME_NUMBER as RESIDENTIAL_ADDRESS_line_1, "
-            + "ura.STREET as RESIDENTIAL_ADDRESS_line_2, "
-            + "ura.CARE_OF as RESIDENTIAL_ADDRESS_CARE_OF, "
-            + "ura.COUNTRY_NAME as RESIDENTIAL_ADDRESS_country, "
-            + "ura.POST_TOWN as RESIDENTIAL_ADDRESS_locality, "
-            + "ura.PO_BOX as RESIDENTIAL_ADDRESS_PO_BOX, "
-            + "ura.POST_CODE as RESIDENTIAL_ADDRESS_POST_CODE, "
-            + "ura.REGION as RESIDENTIAL_ADDRESS_REGION, "
+            + "CASE WHEN OD.SECURE_DIRECTOR_SERVICE_IND = 'Y' THEN NULL ELSE ura.HOUSE_NAME_NUMBER end RESIDENTIAL_ADDRESS_line_1, "
```

Comment:
Just two minor observations:

1. Suggest using lower-case 'od' to be consistent with other query lines.
2. Suggest 'fixing' the column names returned on the lines you've changed to consistently use upper-case rather than a mixture. E.g. 'RESIDENTIAL_ADDRESS_COUNTRY' rather than 'RESIDENTIAL_ADDRESS_country'.